src/MessageReader.js

```diff
@@ -39,22 +39,33 @@ class StandardTypeReader {
     this.view = new DataView(buffer.buffer, buffer.byteOffset);
   }
 
+  json(): { [key: string]: any } {
```

Comment:
```suggestion
  json(): mixed {
```

is the right type, because it's not necessarily an object (it can be a number, array, string, etc too!)